### DISCUSSION TITLE
How do I delete users from the Account API?

### DISCUSSION THREAD

**Discussion by pranauv1** [appwrite/appwrite#5452] [Q&A] [answered]
While the Users API allows deleting users from an admin perspective, is there a way to delete users from the Account API?

_This would be useful in scenarios like "Delete My Account", where the client himself can delete his own account._

**Comment by joeyouss** [comment, marked as answer]
Hi - You can set it up using functions of your own using permissions but I don't think currently we support this internally.

**Reply by stnguyen90** [reply]
To expand on this further, the reason why it's only available server side is because there are probably additional things that need to be deleted (like documents or files) which the Account API wouldn't know about. As mentioned above, you should use the server SDK to delete everything needed including the user.

**Reply by minecrawler** [reply]
For anyone accepting users from the EU, GDPR is a very important topic and I'm actually happy that the Appwrite developer experience makes it harder to forget about personal data outside the user record :)
